tests/services/test_job_execution.py

```diff
@@ -106,19 +106,23 @@ def sync_job_runner_mock():
         yield sync_job_runner_mock
 
 
-def mock_connector(last_sync_status=JobStatus.COMPLETED):
+def mock_connector(job_type=JobType.FULL, last_sync_status=JobStatus.COMPLETED):
```

Comment:
Not a big issue, but I'd rather introduce an additional param `last_access_control_sync_status` to avoid the `if...else...`